### DISCUSSION TITLE
External REST API DataSource in the provider

### DISCUSSION THREAD

**Discussion by waleedshabbir1** [eclipse-edc/Connector#1925] [Q&A]
Hello, 

I can see in the samples there is a  [filetransfer](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/tree/hotfix/milestone_5_1/samples/04.0-file-transfer) sample which works fine.

However, I want to ask, is it possible to get data from an external REST API endpoint instead of transferring files and passing them to the consumer?

**Comment by ndr-brt** [comment]
Hi, sure, currently there's no sample showing that, but the module to be used is `data-plane-http`.
If you'd like to contribute, there's an issue for adding more samples: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1580
